src/middlewared/middlewared/api/v25_04_0/cloud_backup.py

```diff
@@ -31,8 +31,8 @@ class CloudBackupCreate(BaseModel):
     pre_script: LongString = ""
     post_script: LongString = ""
     snapshot: bool = False
-    include: list[NonEmptyString]
-    exclude: list[NonEmptyString]
+    include: list[NonEmptyString] = []
```

Comment:
Should the type annotation be `list | list[NonEmptyString] = []`?

Reply:
An empty list is a valid list of non-empty strings. Non-empty list of non-empty strings would be `Annotated[list[NonEmptyString], Len(min_length=1)]`

Reply:
That doesn't make logical sense. With that logic, this means this is equal `['a']` == `[]`

Reply:
I used `mypy` to statically analyze the annotation and it didn't complain using your annotation or mine so I guess we'll leave this as is but, for the record, claiming `list[NonEmptyString]` is equivalent to an empty list is really strange and not logical imo.